charts/operator-wandb/charts/console/templates/service.yaml

```diff
@@ -11,7 +11,10 @@ metadata:
     {{-   toYaml .Values.service.labels | nindent 4 }}
     {{- end }}
   annotations:
-    {{- include "wandb.deploymentAnnotations" $ | nindent 4 }}
+    {{- if eq .Values.global.cloudProvider "aws" }}
+    alb.ingress.kubernetes.io/healthcheck-path: /console/api/ready
+    {{- end }}
+    {{- include "wandb.serviceAnnotations" $ | nindent 4 }}
```

Comment:
it adds the health check info for the ALB, but previous it was including the `deploymentAnnotations` instead of serviceAnnotations which I assumed was a copy/pase mistake